### DISCUSSION TITLE
第3节课笔记(1班)

### DISCUSSION THREAD

**Discussion by AXYZdong** [InternLM/Tutorial#195] [笔记]
请在下方提交第 3 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/08/ai-Interlm-langchain-RAG/

**Comment by Gu-Xiao** [comment]
https://blog.csdn.net/xidierzuo/article/details/135477856

**Comment by woxinyonghen** [comment]
https://blog.csdn.net/xingchengyonghen/article/details/135466595

**Comment by Xpg74138** [comment]
https://blog.csdn.net/qq_41776136/article/details/135460085

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135481655

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135492002

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135489671

**Comment by RongjianLiang** [comment]
https://zhuanlan.zhihu.com/p/677022214

**Comment by 10086zgyd** [comment]
https://blog.csdn.net/weixin_55546799/article/details/135483176

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/chap2.md

**Comment by strong-leaf** [comment]
大模型(书生.浦语大模型)finetune实战_lesson3笔记与作业 - warm green的文章 - 知乎
https://zhuanlan.zhihu.com/p/677011273

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135489868

**Comment by Andy1314Chen** [comment]
https://www.bilibili.com/read/cv29275301/?jump_opus=1

**Comment by gpt-dance** [comment]
https://zhuanlan.zhihu.com/p/676732671

**Comment by raulming** [comment]
https://quilled-apparatus-d9a.notion.site/505cdf23b70f4bcf8a7eea3b78298bd0?pvs=25

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/KnFwdgJYfoHpiQxiSV7cN5hNnsb?ccm_open_type=lark_docs_home_Recent&from=tab_home#part-EWyUdkmi2oJb1qx7jWUc3cJ1nuc

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/677553043

**Comment by lds-coder** [comment]
https://zhuanlan.zhihu.com/p/677641266

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/677140526
